.ci/scripts/package-docker-snapshot.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+#
+# Build the docker image for the cloudbeat, retag and push it to the given docker registry
```

Comment:
if this does the same as the `mage package` CI job we have we should consider disabling it

Reply:
These scripts are a wrapper around mage commands, to set env-vars/preconditions for mage itself(make release runs mage package).

It makes the pipeline easier to maintain.